### PR TITLE
news about infoflow on risk-v

### PR DESCRIPTION
**Pull request by june-andronick**
Signed-off-by: June Andronick <june@sel4.systems>

### CODE REVIEW

File: content/news/2021.pml [hunk 1]
```diff
@@ -54,6 +54,67 @@ Older News: <%=reflist%>
...
+
+</p>
+<p>
+   The next step in the assurance stack is now also completed for RISC-V with
+   the proof that seL4 enforces confidentiality, i.e. that seL4 provably
+   enforces information flow control, when it is correctly configured to do so.
+</p>
+<p>
+
+   "This complete the 3 big CIA security properties for seL4 on RISC-V:
```
Comment (lsf37) [inline content/news/2021.pml:88]:
```suggestion
   "This completes the 3 big CIA security properties for seL4 on RISC-V:
```

File: content/news/2021.pml [hunk 2]
```diff
@@ -54,6 +54,67 @@ Older News: <%=reflist%>
...
+    <a href="https://riscv.org">
+      <img src="/Foundation/Membership/LOGOS/RISC-V.svg"
+	   style="width: 20%;  padding-left:10px;  float:right"
+	   alt="RISC-V logo" />
+    </a>
+ </p>
+
+<p>
+
+   Last July, we announced that the assurance story for <a
```
Comment (lsf37) [inline content/news/2021.pml:73]:
```suggestion
   In July, we announced that the assurance story for <a
```

**Review by lsf37 — APPROVED** [review]

**Review by gernotheiser — APPROVED** [review]

**Comment by gernotheiser** [comment]
As discussed, please simplify my affiliation to "Scientia Professor Gernot Heiser, chair of the seL4 Foundation" and acknowledge UNSW by "the Trustworthy Systems group at UNSW Sydney". A link to the TS site would also be good

**Comment by june-andronick** [comment]
Link issue seems unrelated and temporary

**Pull request merged**